app/code/community/Bolt/Boltpay/Block/Checkout/Boltpay.php

```diff
@@ -348,7 +348,7 @@ private function getAddressHints($quote, $checkoutType)
         }
 
         // Skip pre-fill for Apple Pay related data.
-        if (!($prefill['email'] == 'fake@email.com' || $prefill['phone'] == '1111111111')) {
+        if (!(@$prefill['email'] == 'fake@email.com' || @$prefill['phone'] == '1111111111')) {
```

Comment:
@matt-thomason  Do you think that this would be better as an **&&**?  Should both conditions necessarily be true to indicate Apple Pay?

Reply:
Either option represents prefilling dummy, non-real data - I think `||` is appropriate.